control-plane/plugin/src/operations.rs

```diff
@@ -1,6 +1,11 @@
-use crate::resources::{utils, CordonResources, DrainResources, GetResources, ScaleResources};
+use crate::resources::{
+    error::Error, utils, CordonResources, DrainResources, GetResources, ScaleResources,
+};
 use async_trait::async_trait;
 
+/// Result wrapper for kubectl commands.
+pub type KubectlResult = Result<(), Error>;
```

Comment:
there's no Kubectl here
```suggestion
/// Result wrapper for plugin commands.
pub type PluginResult = Result<(), Error>;
```